int64 marknumber
float64 x
float64 y
float64 theta
string robotpart

---

bool result
